packages/api/database/seeds/19_CitySeeder.js

```diff
@@ -0,0 +1,51 @@
+/** @type {import('@adonisjs/lucid/src/Lucid/Model')} */
+const City = use('App/Models/City');
+/** @type {import('@adonisjs/lucid/src/Lucid/Model')} */
+const State = use('App/Models/State');
+const fetch = require('node-fetch');
+
+class CitySeeder {
+	constructor() {
+		this.originUrl = 'https://servicodados.ibge.gov.br/api/v1/localidades/distritos';
+	}
+
+	async run() {
+		if ((await City.getCount()) > 0 || (await State.getCount()) > 0) {
+			return;
+		}
```

Comment:
What if any city change it's name or any district get splitted/merged with another one (this may happen soon)?

We'll have to change the code to run the migration again?

Reply:
Wow! Excellent observation!